### DISCUSSION TITLE
Math project

### DISCUSSION THREAD

**Discussion by Boluex** [lukas-blecher/LaTeX-OCR#277] [Q&A]
what free open source ai models that can solve math latex code.....I used pix2tex to turn an image into latex and trying to solve but i don't kow how(using ai and python)....

**Comment by lukas-blecher** [comment]
Hi there,
That depends a bit. What kind of math problems would you like to solve?
You can achieve results with sympy to an extend. For more complicated problems you'll probably need to look at into large language models. The problem is, the good ones are very costly to run.

**Reply by Boluex** [reply]
Okay thanks.....but ive tried sympy and it doesn't work for integration,differentiaton and some latex code generated by the model
